servicex/app/transforms.py

```diff
@@ -138,3 +139,37 @@ async def download_with_progress(filename) -> Path:
 
     for path in result_files:
         print(path.as_posix())
+
+@transforms_app.command(no_args_is_help=True)
+def logs(
+    url: Optional[str] = url_cli_option,
+    backend: Optional[str] = backend_cli_option,
+    transform_id: str = typer.Option(None, "-t", "--transform-id", help="Transform ID"),
+    open_link: Optional[bool] = typer.Option(False, "-o", "--open-link", help="Open Link in browser")
```

Comment:
I'm not sure we need this option. Let's see what people think about it always opening the link

Reply:
Removed the --open-link